src/app/components/popover/index.html

```diff
@@ -1,283 +1,3 @@
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="dismissOnBlur"
-      defaultValue="true"
-      isOptional="true"
-    >
-    Indicates whether to close the popover when it loses focus. To require users to click a trigger button to close the popover, set this input to <sky-code>false</sky-code>. This property accepts <sky-code>boolean</sky-code> values.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="placement"
-      defaultValue="above"
-      isOptional="true"
-    >
-    Specifies the placement of the popover in relation to the trigger element. The <sky-code>skyPopoverPlacement</sky-code> property on the trigger element overwrites this property. The valid options are <sky-code>above</sky-code>, <sky-code>below</sky-code>, <sky-code>left</sky-code>, and <sky-code>right</sky-code>.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="popoverTitle"
-      isOptional="true"
-    >
-    Specifies a title for the popover. This property accepts <sky-code>string</sky-code> values.
-    </sky-demo-page-property>
-  </sky-demo-page-properties>
-
-  <sky-demo-page-properties sectionHeading="Popover events">
-    <sky-demo-page-property
-      propertyName="popoverClosed"
-      isOptional="true"
-    >
-    Fires when users close the popover.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="popoverOpened"
-      isOptional="true"
-    >
-    Fires when users open the popover.
-    </sky-demo-page-property>
-  </sky-demo-page-properties>
-
-  <sky-demo-page-properties sectionHeading="SkyPopoverMessage type">
-    <sky-demo-page-property
-      propertyName="type"
-      isOptional="true">
-      Sets <sky-code>SkyPopoverMessage</sky-code> to one of the following types:
-      <ul>
-        <li>
-          <sky-code>SkyPopoverMessageType.Close</sky-code> &mdash; Closes the popover.
-        </li>
-        <li>
-          <sky-code>SkyPopoverMessageType.Open</sky-code> &mdash; Opens the popover.
-        </li>
-        <li>
-          <sky-code>SkyPopoverMessageType.Reposition</sky-code> &mdash; Repositions the popover to the appropriate position. This is useful for when the popover’s width and height change while it is open.
-        </li>
-      </ul>
-    </sky-demo-page-property>
-  </sky-demo-page-properties>
-
-
-  <sky-demo-page-example>
-    <sky-popover-demo>
-    </sky-popover-demo>
-    <sky-demo-page-code demoName="Popover">
-    </sky-demo-page-code>
-  </sky-demo-page-example>
-
-  </sky-tab>
-
-</sky-tabset>
-
-</sky-tabbed-demo-page>
+<sky-docs-url-redirect
+  url="https://developer.blackbaud.com/skyux-popovers/docs/popover"
```

Comment:
Since we'll eventually have 50 of these, should we factor out as much of the URL in to the component as possible?